ui/src/singletons/store.ts

```diff
@@ -82,158 +87,194 @@ interface AppStore {
 }
 
 // If app becomes too slow, might need to switch to async persistent storage.
-const useStore = create<AppStore>()(
+const useAppStore = create<AppStore>()(
```

Comment:
:question: Why the name change?

Reply:
I was importing useStore from the zustand library which conflicted with the original naming. will change it back since I no longer import it.